portfolio/src/main/webapp/index.html

```diff
@@ -35,21 +35,179 @@ <h1 id="title"> My Portfolio</h1>
+                        <div class="groupContent">
+                            <p>From grades 7 to 12, I commuted to downtown Toronto to attend the University of Toronto Schools (not to be confused with the University of Toronto).
+                                During my last few years of highschool, I took mostly science and math classes, but only took my first coding class in grade 10 which I loved, 
+                                so I decided to participate in coding competitions, become an executive for the school's coding club, and study CS in university. </p>
+                        </div>
+                    </div>
+
+                    <div class="infoGroup">
+                        <div class="groupHeader">
+                            <h3>University</h3>
+                        </div>
+                        <div class="groupContent">
+                            <p>I am currently attending the University of Edinburgh in Scotland and I have finished my first year of my degree in AI and Computer Science.
+                                Outside of classes I also participated in the School of Informatics' coding club and was an active member of Hoppers,
+                                the women in informatics society at the unversity. I had a great time in my first year and am looking forward to the years to come!</p>
+                        </div>
+                    </div>
+
+                    <div class="infoGroup">
+                        <div class="groupHeader">
+                            <h3>QCSYS (2019)</h3>
+                        </div>
+                        <div class="groupContent">
+                            <p>Over the summer last year I was accepted to the Quantum Cryptography School for Young Students at the University of Waterloo. Over the course of two weeks,
+                                I, along with 44 other participants from across the world, learned the basics of quantum mechanics, quantum computing, and quantum cryptography.
+                            </p>
+                        </div>
+                    </div>
                 </div>
 
                 <div title="generalTab"   class="tabContent tabText">             
-                    <h1>GENERAL</h1>   
+                    <h1>GENERAL</h1>
+
+                    <div class="infoGroup">
+                        <div class="groupHeader">
+                            <h3>Intro<h3>
+                        </div>
+                        <div class="groupContent">
+                            <p>My name is Alice Vlasov, I'm 19 years old and I'm from Mississauga, which is a suburb of Toronto in Canada! 
+                                I speak Englich (clearly) and French almost fluently (although my accent is rusty), and also some German and Romanian as well.
+                                My favourite book is A Brief History of Time by Stephen Hawking, and I listen mostly to music from the 80s.<p>
```

Comment:
I liked that book too, although I didn't understand half of it when I read it  😅

Reply:
Hahaha same here, my goal is that one day I'll be able to understand everything 😄